File: Modder/modder/src/main/java/modder/ApkSigner.java
package modder;

import at.favre.tools.apksigner.SignTool;

import java.io.File;

public class ApkSigner {
    static public void Sign(File apkPath){

        String[] args = new String[]{"--apks", apkPath.getAbsolutePath(), "--allowResign", "--overwrite"};
        SignTool.main(args);
    }

}

File: Modder/modder/src/main/java/modder/ApkSigner.kt
package modder

import at.favre.tools.apksigner.SignTool
import java.io.File

class ApkSigner {
    companion object {
        fun Sign(apkPath: File) {
            val args = arrayOf("--apks", apkPath.absolutePath, "--allowResign", "--overwrite")
            SignTool.main(args)
        }
    }
}
